projects/kubernetes-sigs/cluster-api-provider-vsphere/Makefile

```
@@ -15,7 +15,7 @@ EXTRA_GO_LDFLAGS=$(shell $(BUILD_LIB)/version.sh $(REPO) sigs.k8s.io/cluster-api
 
 IMAGE_COMPONENT=kubernetes-sigs/cluster-api-provider-vsphere/release/manager
 
-HAS_S3_ARTIFACTS=true
+HAS_S3_ARTIFACTS=fals
```

```suggestion
HAS_S3_ARTIFACTS=false
```
nit, although technically anything besides "true" is deemed false